client/src/components/BackToTop/BackToTop.stories.tsx

```diff
@@ -1,18 +1,27 @@
+import { Story, Meta } from "@storybook/react";
 import React, { Fragment } from "react";
 
 import { BackToTop } from "./BackToTop";
 
-const BackToTopTemplate = ({ focusId }) => (
+interface TemplateProps {
+  focusId: string;
```

Comment:
Based on the conditional on the input element, it looks like `focusId` was an optional arg in the JS version. Was that wrong?

Reply:
The input element in the JS version also depended on focusId not being null.

Reply:
Right, but that might imply that it was intended to be optional, since the component knew to handle the case where it was falsey (by not showing the input). Was that wrong/unnecessary in the JS version?

Reply:
I think it was intended to be optional there was a `BackToTop` story with the component that depended on `focusId` and one that did not. I made a mistake in defining `focusId` to be required and not optional in the interface.